src/store/actions/actionTypes.js

```diff
@@ -1,5 +1,15 @@
+// import { Component } from "react";
+
 /* eslint-disable linebreak-style */
 export const USER_UI_START_LOADING = 'USER_UI_START_LOADING';
 export const USER_UI_STOP_LOADING = 'USER_UI_STOP_LOADING';
 
 export const SET_USER = 'SET_USER';
+export const FETCH_USER_DATA_REQUEST = 'FETCH_USER_DATA_REQUEST'
+export const FETCH_USER_DATA_SUCCESS = 'FETCH_USER_DATA_SUCCESS'
+export const FETCH_USER_DATA_FAILURE = 'FETCH_USER_DATA_FAILURE'
+
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [More than 1 blank line not allowed.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)